src/ocean/meta/codegen/CTFE.d

```diff
@@ -30,15 +30,15 @@ import ocean.meta.types.Qualifiers;
 
 *******************************************************************************/
 
-istring toString ( long i )
+istring toString ( T ) ( T i )
 {
     if (i == 0)
         return "0";
 
     if (i < 0)
-        return "-" ~ toString(cast(ulong) -i);
+        return "-" ~ toStringImpl(cast(ulong) -i);
 
-    return toString(cast(ulong) i);
+    return toStringImpl(cast(ulong) i);
```

Comment:
I highly doubt this will work. Try:
```
    pragma(msg, toString(int.min), " -- ", int.min);
    pragma(msg, toString(short.min), " -- ", short.min);
```

Reply:
Side note: We should make the `Formatter` CTFE-able to simplify this kind of things

Reply:
It worked but minus sign and cast had to be exchanged :) Added tests for that.